### PR TITLE
fix: Use explicit extension for createMediaQuery import

### LINKED ISSUE #20: Cannot find module on 3.1.0
```
`error - Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/m/Documents/Packagetome/node_modules/use-breakpoint/dist/esm/createMediaQueries' imported from /home/m/Documents/Packagetome/node_modules/use-breakpoint/dist/esm/getCSSMediaQueries.js
    at new NodeError (node:internal/errors:400:5)
    at finalizeResolution (node:internal/modules/esm/resolve:260:11)
    at moduleResolve (node:internal/modules/esm/resolve:879:10)
    at defaultResolve (node:internal/modules/esm/resolve:1087:11)
    at nextResolve (node:internal/modules/esm/loader:161:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:834:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:415:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  digest: undefined
} `
```

```
`{
  packagetome: '0.1.0',
  npm: '9.2.0',
  node: '19.3.0',
  v8: '10.8.168.21-node.8',
  uv: '1.44.2',
  zlib: '1.2.13',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '111',
  nghttp2: '1.51.0',
  napi: '8',
  llhttp: '8.1.0',
  uvwasi: '0.0.13',
  openssl: '3.0.7+quic',
  cldr: '42.0',
  icu: '72.1',
  tz: '2022f',
  unicode: '15.0',
  ngtcp2: '0.8.1',
  nghttp3: '0.7.0'
}`
```

image

### PR DESCRIPTION
**Pull request by ConcernedHobbit**
Fixes #20

### CODE REVIEW

File: src/getCSSMediaQueries.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import type { Config, MediaQuery } from './createMediaQueries'
```
Comment (ConcernedHobbit) [inline src/getCSSMediaQueries.ts:1]: Unused type import MediaQuery removed

**Comment by ConcernedHobbit** [comment]
See 185bc75 for a previous fix to explicit extensions, this follows in line

**Review by matijuguera — APPROVED** [review]
This should fix the issue thx 👍🏻

**Comment by iiroj** [comment]
Thanks for the PR. Should really setup ESLint to complain about this.

**Pull request merged**